torch/_dynamo/skipfiles.py

```diff
@@ -160,7 +160,10 @@ def add(import_name: str):
     if isinstance(import_name, types.ModuleType):
         return add(import_name.__name__)
     assert isinstance(import_name, str)
-    module_spec = importlib.util.find_spec(import_name)
+    try:
+        module_spec = importlib.util.find_spec(import_name)
+    except Exception:
```

Comment:
can you use a more specific except here, like ImportError?

Reply:
I explained at this PR's description. Unfortunately we can't, because ```import torchrec.distributed``` sometime throws ```AttributeError: '_OpNamespace' 'fbgemm' object has no attribute 'jagged_2d_to_dense'```, which is caused by ```fbgemm``` conflicts.

Reply:
hmm, i guess then technically you could catch both 'ImportError' and 'AttributeError' where you check for "'fbgemm' has no attribute" in the string before you return otherwise re-raise?

Reply:
Yes, that's more accurate, let me do that.